### DISCUSSION TITLE
Admin Panel For Serverpod

### DISCUSSION THREAD

**Discussion by rohitsangwan01** [serverpod/serverpod#1436] [Show and tell]
Here is an admin panel for serverpod

https://github.com/rohitsangwan01/serverpod_admin_panel

Easily configurable, and quick setup, and a quick walkthrough video
https://youtu.be/2GDenxG5-3Q?si=5tzg8VKWDilcriOV

**Comment by vlidholt** [comment]
Very cool! Thanks for sharing. I made a post on X too. 🤩

**Reply by rohitsangwan01** [reply]
Oh That’s great, Thank you ☺️

**Comment by sheanie** [comment]
Just tried this package and it looks pretty good.  Good work!

**Reply by rohitsangwan01** [reply]
Thank you ☺️

**Comment by HarrySingh30** [comment]
Really great work 👍. Would be greater if this admin panel was written using flutter itself.

**Reply by donywan** [reply]
> Really great work 👍. Would be greater if this admin panel was written using flutter itself.

Good idea.

**Reply by AdamMusa** [reply]
> Really great work 👍. Would be greater if this admin panel was written using flutter itself.

check that one https://github.com/serverpod/serverpod/discussions/4376#discussion-9211283

**Comment by vborbely** [comment]
> Here is an admin panel for serverpod
> 
> https://github.com/rohitsangwan01/serverpod_admin_panel
> 
> Easily configurable, and quick setup, and a quick walkthrough video https://youtu.be/2GDenxG5-3Q?si=5tzg8VKWDilcriOV

Really cool! That's what I was just looking for!!!

That's a pity it is not written in Dart, but TypeScript. Still promissing.
Do you plan to port it to Dart, as you have the Server&Client YAML description available, so it could be "easily" doable.
What to you think about it @rohitsangwan01 ?

**Reply by AdamMusa** [reply]
> That's a pity it is not written in Dart, but TypeScript. Still promissing.

check that one https://github.com/serverpod/serverpod/discussions/4376#discussion-9211283

**Reply by vborbely** [reply]
Great that you created it! 🙏
